src/js/profile/mobile/widget/Scrollview.js

```diff
@@ -215,7 +215,65 @@
 		],
 		function () {
 			//>>excludeEnd("tauBuildExclude");
-			var Scrollview = ns.widget.core.Scrollview;
+			var Scrollview = ns.widget.core.Scrollview,
+				DOMUtils = ns.util.DOM,
+				classes = Scrollview.classes;
+
+			/**
+			 * Show scroll indicators.
+			 * @method _showScrollIndicator
+			 * @protected
+			 * @member ns.widget.core.Scrollview
+			 */
+			Scrollview.prototype._showScrollIndicator = function () {
+				var self = this,
+					clip = self.element,
+					view = self._ui.view,
+					scrollTop = clip.scrollTop,
+					clipHeight = DOMUtils.getElementHeight(clip),
+					clipOffset = DOMUtils.getElementOffset(clip),
+					viewHeight = DOMUtils.getElementHeight(view),
+					viewWidth = DOMUtils.getElementWidth(view);
+
+				self._clearIndicator(clip);
+
+				switch (self.options.scroll) {
+					case "x":
+					case "xy":
+						// @todo
```

Comment:
Is it planned to be implemented in the next patch?

Reply:
Thank you for review :) . This is good question. This is copy of method from `core/widget/Scrollview.js`. I didn't plan yet when this feature will be implemented. I'll remove `@todo` comment.